oppia_android_test.bzl

```diff
@@ -1,5 +1,9 @@
-load("@rules_jvm_external//:defs.bzl", "artifact")
+"""
+Starlark rules (Macros) to setup the WORKSPACE in the Oppia-Android codebase.
+"""
+
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_android//android:rules.bzl", "android_local_test")
```

Comment:
We actually need to rely on the native android local test, otherwise data-binding isn't going to work with my most recent fix to android_local_test. At the moment, it's a multi-month race between the Android Bazel team and me on whoever is going to fix the new Starlark rule first. :) For now, we need to rely on the native one.

Can we ignore the android rule warning from Buildifier & undo those changes in this PR?

Reply:
Done

Reply:
I actually learned recently that this is a moot point (currently) since the rules_kotlin android_local_test is just a proxy for the native one. That may not hold true indefinitely, though.